src/writer.rs

```diff
@@ -253,12 +286,18 @@ impl<'a> WriterOptions<'a> {
         T: Data,
         D: Dimension,
     {
+        let mut vox_offset: f32 = 352.0;
```

Comment:
Since you had to look this up, a comment to the effect that the vox offset needs to be a float32 per the Nifti spec would help future maintainers.

Reply:
Done, but did it in the structure definition